saleor/graphql/account/mutations/permission_group.py

```diff
@@ -343,7 +361,10 @@ class Meta:
 
     @classmethod
     def clean_instance(cls, info, instance):
-        if not can_user_manage_group(info.context.user, instance):
+        user = info.context.user
```

Comment:
If you're using `requestor` everywhere else, we should use this name here as well for consistency.